code/script.js

```diff
@@ -0,0 +1,72 @@
+//DOM selectors
+const username = 'JensBergqvist'
+const projects = document.getElementById("projects")
+
+
+const API_REPOS = `https://api.github.com/users/${username}/repos`;
+const API_PROFIL = `https://api.github.com/users/${username}`
+//Picture and name
+
+const profilePic = () => {
+    fetch(API_PROFIL)
+        .then(res => res.json())
+        .then(data => {
+           // console.log(data)
+            picture = data.avatar_url
+    let profilePicture =    `<div class = "profile"><h1>${username}</h1> <img class = "photo" src="${picture}" />
+                            </div>`;
+    return (projects.innerHTML = profilePicture)
```

Comment:
Interesting solution with the variable and the the returnstatment, tought me something new!